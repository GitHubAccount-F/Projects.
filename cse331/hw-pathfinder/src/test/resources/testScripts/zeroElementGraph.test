#Tests when the graph contains no elements

#Creates the graph
CreateGraph graph1

#Finds a path, where there shouldn't be any
FindPath graph1 Hero Villian