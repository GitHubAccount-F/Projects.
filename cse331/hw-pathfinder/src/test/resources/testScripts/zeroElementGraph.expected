#Tests when the graph contains no elements

#Creates the graph
created graph graph1

#Finds a path, where there shouldn't be any
unknown: Hero
unknown: Villian